homework/index.js

```diff
@@ -29,23 +31,119 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function renderRepoTable(table, name, path) {
+    const tr = createAndAppend('tr', table);
+    createAndAppend('th', tr, {
+      text: name,
+    });
+    createAndAppend('td', tr, {
+      text: path,
+    });
+    return tr;
   }
 
+  function renderRepoDetails(repo, table) {
```

Comment:
In the HTML you generate you are missing a `table` HTML tag. The parameter name `table` suggests that you are receiving a `table` element through the parameter list, but actually when you call the function you give it a `section` element. 

```html
<section class="repos-container">
  <tr>
    <th>Repository:</th>
    <td>
      <a href="https://github.com/HackYourFuture/alumni" target="_blank"
        >alumni</a
      >
    </td>
  </tr>
  <tr>
    <th>Description:</th>
    <td>Alumni Guides, everything you need to land a job</td>
  </tr>
  <tr>
    <th>Forks:</th>
    <td>9</td>
  </tr>
  <tr>
    <th>Updated:</th>
    <td>10/3/2019, 2:53:12 PM</td>
  </tr>
</section>
```